graduated/codefresh-run/step.yaml

```diff
@@ -144,6 +144,11 @@ spec:
                 "examples": ["33.3", "45"],
                 "default": "45"
             },
+            "TERMINATE_ON_TIMEOUT": {
+                "type": "boolean",
+                "description": "Terminate child build if timeout expired",
```

Comment:
lets rename this to
`Terminate build if timeout reached`

Reply:
✅ Fixed